spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,70 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def calc_tools_spacing(tools_layout):

```

Comment:
I didn't knew that os could differ that much 😲 

I tested in linux with kde and works as expected 😄